### PR TITLE
PMM-3667 create OVF image for pmm2

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: pmm/pmm2-ovf.groovy [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            defaultValue: 'master',
+            description: 'Tag/Branch for percona-images repository',
+            name: 'GIT_BRANCH')
+    }
+    options {
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+    triggers {
+        upstream upstreamProjects: 'pmm-server-autobuild', threshold: hudson.model.Result.SUCCESS
```
Comment (puneet0191) [inline pmm/pmm2-ovf.groovy]: This must be changed to pmm2-server

File: pmm/pmm2-ovf.groovy [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    post {
+        always {
+            deleteDir()
+        }
+        success {
+            script {
+                unstash 'IMAGE'
+                def IMAGE = sh(returnStdout: true, script: "cat IMAGE").trim()
+                slackSend channel: '#pmm-ci', color: '#00FF00', message: "[${specName}]: build finished - ${IMAGE}"
+                slackSend channel: '@ramesh.sivaraman', color: '#00FF00', message: "[${specName}]: build finished - ${IMAGE}"
```
Comment (puneet0191) [inline pmm/pmm2-ovf.groovy]: change this id, to me or Nailya

File: pmm/pmm2-ovf.groovy (hunk 1 above)
Comment (hors) [reply pmm/pmm2-ovf.groovy]: fixed

File: pmm/pmm2-ovf.groovy (hunk 2 above)
Comment (hors) [reply pmm/pmm2-ovf.groovy]: fixed

**Comment by puneet0191** [comment]
Lovely!

**Pull request merged**